### ISSUE TITLE
Not Official

### ISSUE THREAD

**Issue by lucasstark** [wp-premium/woocommerce-dynamic-pricing#1]
Please not this is not approved by the plugin author and is not an official clone.

**Comment by remcotolsma** [comment]
Thanks for your issue, what do you mean by not approved by the plugin author? This is a WordPress plugin with the GPL license, isn't everybody free to share the code?

https://github.com/wp-premium/woocommerce-dynamic-pricing/blob/2.11.0/woocommerce-dynamic-pricing.php

http://www.gnu.org/licenses/gpl-3.0.html

> The licenses for most software and other practical works are designed to take away your freedom to share and change the works. **By contrast, the GNU General Public License is intended to guarantee your freedom to share and change all versions of a program--to make sure it remains free software for all its users.** We, the Free Software Foundation, use the GNU General Public License for most of our software; it applies also to any other work released this way by its authors. You can apply it to your programs, too.

**Comment by lucasstark** [comment]
Yes, while the code is 100% GPL, this repository is not a fork and contains copyrighted work, such as the Plugin Name and contents of readme.txt files and the plugin description.   

Repositories and exact clones like this also cause issues on the support side because someone needing assistance won't be part of the official support channels and often get confused when seeking help.

**Comment by remcotolsma** [comment]
I'm no expert in this, but i thought GPL allows everybody the share a exact copy of the plugin?
